lightning/src/ln/features.rs

```diff
@@ -395,6 +399,68 @@ impl InvoiceFeatures {
 	}
 }
 
+impl ToBase32 for InvoiceFeatures {
+	fn write_base32<W: WriteBase32>(&self, writer: &mut W) -> Result<(), <W as WriteBase32>::Err> {
```

Comment:
Could you add some tests for these conversions?

Reply:
There is `test_payment_secret_and_features_de_and_ser` which uses those methods, and invoice fuzzing. Added more unit tests though.